contrib/babelfishpg_tsql/src/hooks.c

```diff
@@ -408,6 +414,9 @@ InstallExtendedHooks(void)
 
 	pre_exec_tsql_cast_value_hook = exec_tsql_cast_value_hook;
 	exec_tsql_cast_value_hook = pltsql_exec_tsql_cast_value;
+
+	InitializeParallelDSM_hook = babelfixedparallelstate_insert;
```

Comment:
let us add tsql or bbf prefix for these hook names

Reply:
Updated in latest rev.